content/en/docs/tasks/administer-cluster/controller-manager-leader-migration.md

```diff
@@ -0,0 +1,109 @@
+---
+reviewers:
+- jpbetz
+- cheftako
+title: "Controller Manager Leader Migration"
+linkTitle: "Controller Manager Leader Migration"
```

Comment:
Possibly “Migrate A Replicated Control Plane To Use The cloud-controller-manager”?

Reply:
I simplified it a bit to "Migrate Replicated Control Plane To Use Cloud Controller Manager", matching titles of existing pages. What do you think?